distribution/changelog.txt

```diff
@@ -8,6 +8,7 @@
 - Change: [#22740] Add virtual floor to shifted track design placement.
 - Change: [#22795] Replace Giga Coaster and Twister Roller Coaster booster images.
 - Fix: [#2614] The colour tab of the ride window does not hide invisible cars (original bug).
+- Fix: [#7672] Wide path status is set to all 'wide' paths, instead of only a quarter, impeding pathfinding.
```

Comment:
Quickly, replace these with typewriter quotes before @Gymnasiast finds out.
```suggestion
- Fix: [#7672] Wide path status is set to all ‘wide’ paths, instead of only a quarter, impeding pathfinding.
```